capsule/src/main/java/Capsule.java

```diff
@@ -2470,6 +2454,65 @@ private void resolveNativeDependencies() {
         return new ArrayList<String>(jvmArgs.values());
     }
 
+    private List<String> parseCommandLineArguments(String toProcess) {
```

Comment:
A comment would do. Write something like: "this code is a modified version of ..."